README.md

```diff
@@ -929,6 +929,8 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Imitate Email](https://imitate.email) - Sandbox Email Server for testing email functionality across build/qa and ci/cd. Free accounts get 15 emails a day forever.
   * [ImprovMX](https://improvmx.com) – Free email forwarding.
   * [EForw](https://www.eforw.com) – Free email forwarding for one domain. Receive and send emails from your domain.
+  * [Inboxes App](https://inboxesapp.com) — Create up to 3 temporary emails a day, then delete them when you're done from within a handy Chrome extension. Perfect for testing signup flows.
+     - [SPF, DKIM and DMARC check tool](https://inboxesapp.com/simple-dmarc-spf-dkim-checker/) Quickly check that SPF, DKIM and DMARC is set up correctly for free.
```

Comment:
We should either combine this to one line or pick what we're adding, we are trying to limit the multi line entries and we only list the same app once

Reply:
@ripienaar This has been updated and is now ready for your review and consideration. Thanks again.